ckanext/canada/commands.py

```diff
@@ -352,9 +352,12 @@ def _changed_packages_since(self, registry, since_time):
         packages = []
         for result in data:
             package_id = result['package_id']
-            source_package = registry.action.package_show(id=package_id)
-            source_package = get_datastore_and_views(source_package, registry)
-            packages.append(json.dumps(source_package))
+            try:
+              source_package = registry.action.package_show(id=package_id)
+              source_package = get_datastore_and_views(source_package, registry)
+              packages.append(json.dumps(source_package))
+            except NotFound:
+              print package_id + " not found in database."
```

Comment:
```suggestion
            try:
                source_package = registry.action.package_show(id=package_id)
            except NotFound:
                print package_id + " not found in database."
            else:
                source_package = get_datastore_and_views(source_package, registry)
                packages.append(json.dumps(source_package))
```

Reply:
*always* 4 space indents for python files and using `else` here lets us be more precise with where the exception is coming from

Reply:
@wardi oops, my editorconfig in that project was set to 2 indents, that is fixed now. And I added the `else` statement as well.